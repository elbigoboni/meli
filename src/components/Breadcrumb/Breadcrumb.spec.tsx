import { render } from "@testing-library/react";
import React from "react";
import { Breadcrumb } from ".";

describe("<Breadcrumb />", () => {
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(
        <Breadcrumb
          path={[
            "EletrÃ´nica, Audio y Video",
            "iPod",
            "Reproductores",
            "iPod touch",
            "32gb",
          ]}
        />
      );
      expect(container).toMatchSnapshot();
    });
  });
});
