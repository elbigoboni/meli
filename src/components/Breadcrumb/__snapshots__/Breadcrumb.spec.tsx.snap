// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> Render should render correctly 1`] = `
<div>
  <div
    class="sc-bdfBwQ cgnLdY"
  >
    Eletrônica, Audio y Video › iPod › Reproductores › iPod touch
     
     › 
     
    <em
      class="sc-gsTCUz bCQwq"
    >
      32gb
    </em>
  </div>
</div>
`;
